### DISCUSSION TITLE
Do we have any separate model for Circuit Breaker Or Switches

### DISCUSSION THREAD

**Discussion by SanjeevKumarGoyal** [CURENT/andes#323] [Modeling]
Good Morning Dr. Cui,
Just want to make sure about the separate model for `Circuit Breaker` Or `Switches`.
Your response in this regards will be very helpful.

Thank you very much!

Regards,
Sanjeev

**Comment by cuihantao** [comment]
The model we have is Toggler, which can be used to change the connectivity status of a line.

We don’t have a node-breaker model so strictly we don’t have a circuit breaker.

Regards,
Hantao Cui
